web/themes/custom/novel/novel.theme

```diff
@@ -146,4 +147,55 @@ function novel_form_system_theme_settings_alter(array &$form, FormStateInterface
   // Remove the Path field from the logo and favicon upload fields.
   unset($form['logo']['settings']['logo_path']);
   unset($form['favicon']['settings']['favicon_path']);
+  // Add in identity settings.
+  $form['identity']['settings'] = [
+    '#type' => 'fieldset',
+    '#title' => t('identity settings'),
+  ];
+  $form['identity']['settings']['identity_color'] = [
+    '#type' => 'color',
+    '#title' => t('Identity Color', [], ['context' => 'Identity settings']),
+    '#default_value' => theme_get_setting('identity_color'),
+    '#description' => t('Choose library identity color.', [], ['context' => 'Identity settings']),
```

Comment:
Consider changing the translation context to "Novel settings" or the like

Now that we have moved the implementation the context could move as well. By using the module or theme name as the context we avoid having a million different contexts.